locales/en.yaml

```diff
@@ -21,84 +21,141 @@ system:
 modules:
   telegram:
     chatMigration:
-      notification: |
-        {{.Name}} @{{.Username}} 监测到您的群组已从 <b>群组（group）</b> 升级为了 <b>超级群组（supergroup）</b>，届时，群组的 ID 将会发生变更，<b>现已自动将过去的历史记录和数据留存自动迁移到了新的群组 ID 名下</b>，之前的设置将会保留并继续沿用，不过需要注意的是，由于 Telegram 官方的限制，迁移事件前的消息 ID 将无法与今后发送的消息 ID 相兼容，所以当下一次总结消息时将不会包含在迁移事件发生前所发送的消息，由此带来的不便敬请谅解。
+  notification: |
+    {{.Name}} @{{.Username}} has upgraded your group to a <b>supergroup</b>, resulting in a change of group ID. <b>We have automatically migrated all historical records and data to the new group ID</b>. All previous settings will remain in effect. However, due to Telegram's restrictions, message IDs from before the migration won't match those sent afterwards. This means messages sent before the upgrade won't be included in future summaries. We apologize for any inconvenience.
```

Comment:
```
{{.Name}} @{{.Username}} has upgraded
```

Should not be `has upgraded`, shouldn't it be `has detected that your group has been upgraded to`?

Reply:
```
All previous settings will remain in effect.
```

`effect`?